reference/pointset.md

```diff
@@ -0,0 +1,22 @@
+## PointSet
+
+```
+PointSet {
+  SFNode  color   NULL  # {Color, PROTO}
+  SFNode  coord   NULL  # {Coordinate, PROTO}
+}
+```
+
+### Description
+
+The [PointSet](#pointset) node represents a 3D geometry formed by a set of 3D points specified in the `coord` field.
+[PointSet](#pointset) nodes are not lit, are not texture-mapped and they do not cast or receive shadows.
+[PointSet](#pointset) nodes cannot be used for collision detection (boundingObject).
+
+### Field Summary
+
+The `color` field optionally contains a [Color](color.md) node which defines the color of each point.
+If the `color` field is not NULL, the [Color](color.md) node should have the same number of items than the [Coordinate](coordinate.md) node in the `coord` field.
+If the `color` field is NULL, the `emissiveColor` of the [Material](material.md) will be used to define the color of the points. 
+
+The `coord` field specifies the list of 3D points and contains a [Coordinate](coordinate.md) node.
```

Comment:
I would say: "The `coord` field contains a [Coordinate](coordinate.md) node that specifies the list of 3D points."